modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/common/LoggingAdminClient.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
We can bring this class to a more common place than this. This is saml common. This might be needed for other tests as well.